app/models/distribution.rb

```diff
@@ -49,7 +51,11 @@ class Distribution < ApplicationRecord
   delegate :name, to: :partner, prefix: true
 
   def distributed_at
-    issued_at.strftime("%B %-d %Y")
+    if is_midnight(issued_at)
+      issued_at.strftime("%B %-d %Y")
+    else
+      issued_at.to_s(:distribution_date_time)
+    end
```

Comment:
I'm not sure this is necessary -- what's the case this is covering?

Reply:
For new `issued_at` date times that have non-zero hours and minutes, display the hour:minute with am/pm.

For old, or new but purposefully empty time, `issued_at` date times, display the date in the old way. This keeps those records displaying as before without startling the viewer with "March 31 2019 12:00am

Reply:
:+1: